extensions/reviewed/Bounce.json

```
@@ -469,6 +492,47 @@
             }
           ],
           "objectGroups": []
+        },
+        {
+          "description": "the number of times this object has bounced off another object of the object.",
+          "fullName": "Number of times this object has bounced off another object",
```

The name must be as short as possible to allow user to scan them.
```suggestion
          "fullName": "Bounce count",
```